lib/src/util/react_util.dart

```diff
@@ -26,13 +27,13 @@ import 'package:react/react_client.dart';
 /// DEPRECATED: Use new boilerplate mixin pattern instead (see the New Boilerplate Migration
 /// Guide for more information).
 @Deprecated('This pattern is deprecated in favor of the mixin props mixin pattern. See the New Boilerplate Migration guide for more information.')
-class UiPropsMapView extends MapView
+abstract class UiPropsMapView extends MapView
     with
         ReactPropsMixin,
         UbiquitousDomPropsMixin,
         CssClassPropsMixin
     implements
-        component_base.UiProps {
+        UiProps {
```

Comment:
Implementing the generated UiProps makes it so that we can use `@override` in the methods we're overriding below; not sure why we weren't always doing this.